src/map/Lines.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "./Layer", "css!./Lines"], factory);
+    } else {
+        root.map_Graph = factory(root.d3, root.map_Layer);
+    }
+}(this, function (d3, Layer) {
+    function Lines() {
+        Layer.call(this);
+    }
+    Lines.prototype = Object.create(Layer.prototype);
+    Lines.prototype.constructor = Lines;
+    Lines.prototype._class += " map_Lines";
+
+    Lines.prototype.publish("opacity", 1.0, "number", "Opacity", null, { tags: ["Advanced"] });
+
+    Lines.prototype.data = function (_) {
+        var retVal = Layer.prototype.data.apply(this, arguments);
+        if (arguments.length) {
+            this.dataEdges = [];
+            _.forEach(function (row) {
+                this.dataEdges.push({
+                    type: "LineString",
+                    coordinates: [[row[1], row[0]], [row[3], row[2]]]
+                });
+            }, this);
+        }
+        return retVal;
+    };
+
+    Lines.prototype.layerEnter = function (base, svgElement, domElement) {
+        Layer.prototype.layerEnter.apply(this, arguments);
+
+        svgElement.append("defs").append("marker")
+            .attr("class", "marker")
+            .attr("id", this._id + "_arrowHead")
+            .attr("viewBox", "0 0 10 10")
+            .attr("refX", 10)
```

Comment:
I am going to leave this as a 1.14 addition.  The main reason being that coloring the Arrow Heads is somewhat non trivial (since you have to define a new one for each color) and I would want to common that code up with the Graph Widget